llvm/tools/llvm-exegesis/lib/LlvmState.cpp

```diff
@@ -93,7 +93,7 @@ LLVMState::LLVMState(std::unique_ptr<const TargetMachine> TM,
 std::unique_ptr<TargetMachine> LLVMState::createTargetMachine() const {
   return std::unique_ptr<TargetMachine>(
       TheTargetMachine->getTarget().createTargetMachine(
-          TheTargetMachine->getTargetTriple().normalize(),
+          Triple(TheTargetMachine->getTargetTriple().normalize()),
```

Comment:
```suggestion
          TheTargetMachine->getTargetTriple(),
```

Reply:
I tried that initially, but this one seems to be actually important (there are test failures otherwise).